src/UnityExtension/Assets/Editor/GitHub.Unity/UI/GitPathView.cs

```diff
@@ -0,0 +1,286 @@
+        }
+
+        public override bool IsBusy
+        {
+            get { return isBusy; }
+        }
+
+        public override void OnGUI()
+        {
+            // Install path
+            GUILayout.Label(GitInstallTitle, EditorStyles.boldLabel);
+
+            EditorGUI.BeginDisabledGroup(IsBusy || Parent.IsBusy);
+            {
+                // Install path field
+                GUILayout.BeginHorizontal();
+                {
+                    EditorGUI.BeginChangeCheck();
+                    {
+                        newGitExec = EditorGUILayout.TextField(PathToGit, newGitExec);
+                    }
+
+                    if (EditorGUI.EndChangeCheck())
+                    {
+                        CheckEnteredGitPath();
+                    }
+
+                    if (GUILayout.Button(BrowseButton, EditorStyles.miniButton, GUILayout.Width(25)))
+                    {
+                        GUI.FocusControl(null);
+
+                        var newValue = EditorUtility.OpenFilePanel(GitInstallBrowseTitle,
+                            gitExecParent,
+                            gitExecExtension);
+
+                        if (!string.IsNullOrEmpty(newValue))
+                        {
+                            newGitExec = newValue;
+
+                            if (Environment.IsWindows)
+                            {
+                                //Normalizing the path separator in windows
+                                newGitExec = newGitExec.ToNPath().ToString();
+                            }
+
+                            CheckEnteredGitPath();
+                        }
+                    }
+                }
+                GUILayout.EndHorizontal();
+
+                GUILayout.Space(EditorGUIUtility.standardVerticalSpacing);
+
+                GUILayout.BeginHorizontal();
+                {
+                    EditorGUI.BeginDisabledGroup(!isValueChangedAndFileExists);
+                    {
+                        if (GUILayout.Button(GitPathSaveButton, GUILayout.ExpandWidth(false)))
+                        {
+                            GUI.FocusControl(null);
+                            isBusy = true;
+
+                            ValidateAndSetGitInstallPath(newGitExec);
+                        }
+                    }
+                    EditorGUI.EndDisabledGroup();
+
+                    //Find button - for attempting to locate a new install
+                    if (GUILayout.Button(GitInstallFindButton, GUILayout.ExpandWidth(false)))
+                    {
+                        GUI.FocusControl(null);
+                        isBusy = true;
+
+                        newGitExec = gitExec;
+                        CheckEnteredGitPath();
+
+                        new ProcessTask<NPath>(Manager.CancellationToken, new FirstLineIsPathOutputProcessor())
+                            .Configure(Manager.ProcessManager, Environment.IsWindows ? "where" : "which", "git")
```

Comment:
`FindExecTask` that does this type of executable location via where/which

Reply:
👍